lib/input_sanitizer/version.rb

```diff
@@ -1,3 +1,3 @@
 module InputSanitizer
-  VERSION = "0.3.27"
```

Comment:
bump version in other commit/pr - our current best practice is to add features and then release the features in a saparate step